src/Tempest/Http/src/Routing/Construction/RouteTreeNodeType.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Http\Routing\Construction;
+
+enum RouteTreeNodeType
```

Comment:
Pure on intuition, I'd assume here to be `Dynamic` as well? It's probably not necessary but I figured I'd ask to be sure

Reply:
I have renamed `Parameter` as dynamic. you were right I wasn't consistent in my naming